Add warning for known issue in Zeebe 1.3.0 (#562)
camunda-cloud/camunda-cloud-documentation: docs/guides/update-guide/120-to-130.md

## Server

### Zeebe

:::caution
The docker image for Zeebe 1.3.0 does not set a system locale.

> This can lead to user data corruption, in particular with non-ascii characters (such as umlauts) in process variables.

Reviewer: ```suggestion
This can lead to problems with user data, like process variables, which use non-ASCII characters (such as umlauts).
```

My proposal is to not call this corruption, as this is makes users quite uncomfortable, wdyt?

Reviewer: we agreed it is data corruption
